packages/core/components/DropZone/styles.module.css

```diff
@@ -0,0 +1,61 @@
+.DropZone {
+  margin-left: auto;
+  margin-right: auto;
+  zoom: 1.33;
+  position: relative;
+  height: 100%;
+  outline-offset: -1;
+  width: 100%;
+}
+
+.DropZone-content {
+  min-height: 128px;
```

Comment:
Not using a system, I guess it felt futile since the vertical rhythm is defined by the user's components.

There's also the zoom factor.

Not opposed to using one regardless for consistency

Reply:
Sure, one for #later